chaos/create_actor_failures.go

```diff
@@ -0,0 +1,23 @@
+package chaos
+
+import (
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	"github.com/filecoin-project/specs-actors/actors/runtime"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+	"github.com/ipfs/go-cid"
+)
+
+func (a Actor) CreateAccountActorWithAddr(rt runtime.Runtime, addr *address.Address) *adt.EmptyValue {
+	rt.CreateActor(builtin.AccountActorCodeID, *addr)
+
+	return nil
+}
+
+func (a Actor) CreateUnknownActor(rt runtime.Runtime, addr *address.Address) *adt.EmptyValue {
```

Comment:
Confirmed we get the expected WARN from the VM:
```
2020-08-20T17:19:51.732+0200    WARN    vm      vm/runtime.go:303       Abortf: Can only create built-in actors.
2020-08-20T17:19:51.732+0200    WARN    vm      vm/runtime.go:116       VM.Call failure: Can only create built-in actors. (RetCode=10):
    github.com/filecoin-project/lotus/chain/vm.(*Runtime).CreateActor
        /Users/nonsense/code/pkg/mod/github.com/filecoin-project/lotus@v0.4.3-0.20200819134055-b13681df3205/chain/vm/runtime.go:225
```

Reply:
Nit: I would export a method `CreateActor` with params matching those passed to `rt.CreateActor`, so that the caller can invoke the rt method however they want. I think this is more flexible that exporting a new method for each invalid way of invoking the method.

(Note that you won't be able to use Cid.Undef, though, since it doesn't serialize)